Contrib/CCCS/AlgebraicLiteracy/IA_9.5/OpenStax_IA_9.5_216.pg

```diff
@@ -33,15 +33,16 @@ $showPartialCorrectAnswers = 1;
 #  Setup
 
 Context()->variables->add(t=>"Real");
-Context()->flags->set(
-  tolerance => 0.0051,
-  tolType => "absolute",
-);
-
 
 $v0 = random(150, 300, 10);
 $h0 = random(3, 6);
-$t = Compute("(-$v0 - sqrt($v0^2 + 64*$h0))/-32");
+
+$t = Compute("(-$v0 - sqrt($v0**2 + 64*$h0))/-32");
+$t = Round($t,2);
+$t = Compute ($t)->with(
+tolType => 'absolute',
+tolerance => 0.0005,
```

Comment:
This will require accuracy to the thousandth of a second.

Reply:
I was under the impression this was a 'better safe than sorry' thing because this is the tolerance and the rounding is in the line above. But I will correct this to have tolerance = > 0.005,

Reply:
I see -- so you're enforcing that the rounding took place?

Reply:
Correct. Ensuring that any other rounding would be marked incorrect, although perhaps this is not necessary.

Reply:
No, probably not necessary -- but it does enforce the instructions in the problem. Since that is what you intended, there's no need to change it (in a way that would accept answers that are not rounded according to the instructions).